stubs/core.stub

```diff
@@ -296,3 +296,8 @@ function headers_sent(string &$filename = null, int &$line = null): bool {}
  * @param-out array<string, string|array<string>> $result
  */
 function mb_parse_str(string $string, &$result = []): bool {}
+
+/**
+ * @param-out callable|false $callable_name
```

Comment:
Should rather be callable-string. And have you verified what's set in case of non-callable being passed in?

Reply:
yep, I did.

https://3v4l.org/2UpJB

Reply:
So it's never false?

What about if you pass in a nonexistent method via the array syntax?

Reply:
Ahh right: it doesn't get false

https://3v4l.org/HctjS